### PR TITLE
Add Fedora 24 beta to packer definitions

### PR DESCRIPTION
**Pull request by stahnma**

### CODE REVIEW

File: templates/fedora-24/i386.vmware.base.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+
+  "variables":
+    {
+      "template_name": "fedora-24-i386",
+      "template_os": "fedora",
+
+      "iso_url": "http://osmirror.delivery.puppetlabs.net/iso/Fedora-Server-dvd-i386-24_Beta-1.6.iso"
```
Comment (heathseals) [inline templates/fedora-24/i386.vmware.base.json]: missing a comma at the end of this line.

**Comment by shermdog** [comment]
Does it blend?

**Comment by stahnma** [comment]
I *think* kickstart is currently broken with Fedora 24.

**Comment by stahnma** [comment]
This has been obsoleted by #77

**Pull request closed without merging**